core/indexer/elasticsearchDatabase_test.go

```diff
@@ -558,6 +559,74 @@ func TestIndexTransactionDestinationBeforeSourceShard(t *testing.T) {
 	esDatabase.SaveTransactions(body, header, txPool, 0)
 }
 
+func TestDoBulkRequestLimit(t *testing.T) {
+	t.Skip("test must run only if you have an elasticsearch server on address http://localhost:9200")
+
+	args := elasticSearchDatabaseArgs{
+		url:                    "http://localhost:9200",
+		userName:               "basic_auth_username",
+		password:               "basic_auth_password",
+		marshalizer:            &mock.MarshalizerMock{},
+		hasher:                 &mock.HasherMock{},
+		addressPubkeyConverter: &mock.PubkeyConverterMock{},
+	}
+
+	esDatabase, _ := newElasticSearchDatabase(args)
+
+	//Generate transaction and hashes
+	numTransactions := 1000
+	dataSize := 123546
+	txs, hashes := generateTransactions(numTransactions, dataSize)
+
+	header := &dataBlock.Header{}
+	txsPool := make(map[string]data.TransactionHandler)
+	for i := 0; i < numTransactions; i++ {
+		txsPool[hashes[i]] = &txs[i]
+	}
+
+	miniblock := &dataBlock.MiniBlock{
+		TxHashes: make([][]byte, numTransactions),
+		Type:     dataBlock.TxBlock,
+	}
+	for i := 0; i < numTransactions; i++ {
+		miniblock.TxHashes[i] = []byte(hashes[i])
+	}
+
+	body := &dataBlock.Body{
+		MiniBlocks: []*dataBlock.MiniBlock{
+			miniblock,
+		},
+	}
+	body.MiniBlocks[0].ReceiverShardID = 2
+	body.MiniBlocks[0].SenderShardID = 1
+	esDatabase.SaveTransactions(body, header, txsPool, 2)
```

Comment:
There is no assert here

Reply:
yes this test does not have assert because is more like a manual test and i run it when i want to check if transactions are saved in elasticsearch database